src/test/functional/Superescalar/loop.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { expect, beforeEach, test } from 'vitest'
+import { Code } from '../../../core/Common/Code';
+import { Superescalar } from '../../../core/Superescalar/Superescalar';
+import { SuperescalarStatus } from '../../../core/Superescalar/SuperescalarEnums';
+import { codeInput, vecContent, sumContent, resultContent } from "../code/bucle";
+import { codeInput as doubleCodeInput } from "../code/bucledoble";
+import { codeInput as softCodeInput } from "../code/buclesoft";
+
+
+const context: { code: Code, machine: Superescalar } = { code: null, machine: null };
+
+beforeEach(() => {
+    context.code = new Code();
+    context.machine = new Superescalar();
+    context.machine.init(true);
+});
+
+test('Bucle.pla is executed properly', t => {
+    // Load code
+    context.code.load(codeInput);
+    context.machine.code = context.code;
+
+    // Load memory
+    context.machine.memory.setDatum(40, sumContent);
+    for (let i = 50; i < vecContent.length + 50; i++) {
+        context.machine.memory.setDatum(i, vecContent[i - 50]);
+    }
```

Comment:
Seems a bit convoluted
```
for (let i = 0; i < vecContent.length; i++) {
    context.machine.memory.setDatum(i + 50, vecContent[i]);
}
```

Reply:
There are more instances of this, look for them!

Reply:
Soryy, I read this fast and I thought you wanted me to do this in another way, for example with `splice`:
```
const vecBaseAddress = 50;
    context.machine.memory.data.splice(vecBaseAddress, vecContent.length, ...vecContent);
```

I think this way is more elegant than my older aproach, if you also like it, I will use this way.

Reply:
LGTM 👍🏼